ui/Screen/Org/Projects.svelte

```diff
@@ -2,54 +2,81 @@
   import { push } from "svelte-spa-router";
 
   import * as path from "../../src/path.ts";
-
-  import { Text, Button } from "../../DesignSystem/Primitive";
-  import Placeholder from "../../DesignSystem/Component/Placeholder.svelte";
+  import { Flex, Icon, Text } from "../../DesignSystem/Primitive";
+  import {
+    AdditionalActionsDropdown,
+    List,
+    ProjectCard,
+    Remote,
+    Stats,
+  } from "../../DesignSystem/Component";
+  import { orgs as orgsStore, fetchProjectList } from "../../src/org.ts";
 
   export let params = null;
 
-  const registerProject = () => push(path.registerProject(params.id));
-  const registerMember = () => push(path.memberRegistration(params.id));
+  const select = (event) => {
+    const orgProject = event.detail;
+    if (orgProject.maybeProject) {
+      push(path.projectSource(orgProject.maybeProject.id));
+    }
+  };
+
+  $: fetchProjectList({ id: params.id });
+
+  const statsProps = (stats) => {
+    return [
+      { icon: Icon.Commit, count: stats.commits },
+      { icon: Icon.Branch, count: stats.branches },
+      { icon: Icon.Member, count: stats.contributors },
+    ];
+  };
+
+  const projectCardProps = (orgProject) => {
+    if (orgProject.maybeProject) {
+      return {
+        title: orgProject.name,
+        description: orgProject.maybeProject.metadata.description,
+        showRegisteredBadge: true,
+      };
+    } else {
+      return {
+        title: orgProject.name,
+        showRegisteredBadge: true,
+      };
+    }
+  };
 </script>
 
-<style>
-  .wrapper {
-    margin-top: 156px;
-    display: flex;
-    justify-content: center;
-    align-items: center;
-  }
-
-  .content {
-    width: 420px;
-  }
-
-  .buttons {
-    display: flex;
-    justify-content: center;
-  }
-</style>
-
-<div class="wrapper" data-cy="projects">
-  <div class="content">
-    <Placeholder style="width: 100%; height: 217px; margin-bottom: 24px;" />
-    <Text style="margin-bottom: 24px;">
-      There's nothing here yet. Get started by creating your first project or
-      adding a member to your organization.
-    </Text>
-    <div class="buttons">
-      <Button
-        variant="vanilla"
-        style="margin-right: 25px;"
-        on:click={registerProject}>
-        Register a project
-      </Button>
-      <Button
-        variant="vanilla"
-        on:click={registerMember}
-        dataCy="add-member-button">
-        Add a member
-      </Button>
-    </div>
+<Remote store={orgsStore} let:data={orgProjects}>
+  {#if orgProjects.length > 0}
+    <List items={orgProjects} on:select={select} let:item={orgProject}>
+      {#if orgProject.maybeProject}
```

Comment:
Shouldn't we move this out into its own component e.g. `OrgProjectCard`, which can wrap `ProjectCard` and knows how to render itself depending if a coco project is present.

Reply:
We can move it to a wrapper if there is more than one place where we need it.